opl/http.py

```diff
@@ -8,6 +8,12 @@
 session = requests.Session()
 
 
+def insecure():
+    session.verify = False
+    logging.debug("Disabling insecure request warnings")
```

Comment:
This is not needed, it will be printed in that `disable_insecure_request_warnings`. Or maybe you meant:
```suggestion
    logging.debug("Disabling SSL verifications for this session")
```